protocols/medco/deterministic_switching_test.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func TestDeterministicSwitching(t *testing.T) {
+	// Issue with deadlock occurences in the medco protocol
+	// see https://github.com/dedis/cothority/issues/479
+	t.Skip()
```

Comment:
Put the issue in the skip itself:
`t.Skip("Issue #479 with deadlock occurences in the medco protocol")`